.env

```diff
@@ -0,0 +1 @@
+API_URL='http://localhost:3000'
```

Comment:
Hey man you really got to start gitignoring this. I'm going to have to take a point off for this.